pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -108,9 +114,15 @@ func isBindtoPort(l *listener.Listener) bool {
 	return bp.Value
 }
 
+// enabledInspector captures if a given listener needs listener filter inspectors added
```

Comment:
nit: enabledInspector captures if for a given listener, listener filter inspectors are added